babel.config.js

```diff
@@ -8,7 +8,8 @@ module.exports = api => {
   if (api && api.env('test')) {
     presets.push('@babel/preset-typescript')
     plugins.push(['@babel/plugin-proposal-class-properties', { loose: true }])
-    plugins.push(['@babel/plugin-transform-modules-commonjs'])
+    plugins.push('@babel/plugin-transform-modules-commonjs')
+    plugins.push('@babel/plugin-proposal-optional-chaining')
```

Comment:
allows optional chaining in tests e.g. `expect(err?.meesage).toBe(` which is what the lint rules recommend